README.md

```diff
@@ -152,11 +152,13 @@ Measured::Weight.unit_names_with_aliases
   * oz, ounce, ounces
 * `Measured::Length`
   * m, meter, metre, meters, metres
+  * km, kilometer, kilometre, kilometers, kilometres
```

Comment:
We could perhaps document what the SI prefixes are earlier in the `README`, because we now have `si_unit` as a helper for building a unit system, which should also be documented 😅